src/store/reducers/Authentication/index.test.ts

```diff
@@ -88,7 +86,7 @@ describe('auth slice', () => {
         expect(errorAction.payload).toEqual({ data: mockError.response?.data, status: mockError.response?.status });
         expect(errorAction.meta.arg).toBe(input);
 
-        expect(dispatch).toHaveBeenNthCalledWith(1, signIn.pending(errorAction.meta.requestId, input));
+        expect(dispatch).toHaveBeenNthCalledWith(1, signInAsyncThunk.pending(errorAction.meta.requestId, input));
```

Comment:
Why don't you use await or Promise resolve in the function and assert the result (returned from success/failed cases) 🤔. That's meaningless if we just checking the function is called in the pending regardless checking the final results

Reply:
@nvminhtue ah I got it. It is unnecessary to verify the call here. Let me verify the result instead.

Reply:
@nvminhtue I updated the verify's part in [3aadb5c](https://github.com/manh-t/react-survey/pull/56/commits/3aadb5c4ed29ded049fb3ae88f102051848a7681)